apps/portal/src/content/docs/components/button-layout.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: ButtonLayout
+description: ButtonLayout component
+beta: true
+---
+
+The `ButtonLayout` component allows you to group multiple buttons together, either horizontally or vertically. It is useful for organizing related actions.
+
+import { DocsPage } from "../../../components/docs-page";
+
+## Default `ButtonLayout` with `horizontal` orientation and `end` align
```

Comment:
Let's have just a single example at the top that shows the typical use-case. We can then have sections after the "Usage" section. e.g. https://harness-design.netlify.app/components/card-select/